src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -480,17 +443,154 @@ public void ValidateXml(string schemaFile, ValidationEventHandler validationEven
         /// Gets the root element of the current part.
         /// Returns null when the current part is empty or is not an XML content type.
         /// </summary>
-        public OpenXmlPartRootElement? RootElement
+        public OpenXmlPartRootElement? RootElement => PartRootElement;
+
+#endregion
+
+#region Linq to XML
```

Comment:
Could you create a new file for a partial class of OpenXmlPart.Linq? These class are getting too large and I'm not a fan of regions

Reply:
The `OpenXmlPart` class has not been split into partial classes. I added the Linq region because there were already 9 other regions in there. The Linq feature accounts for 140 of 908 line items. Do you really want to do this?